content/data-stories/traffic-and-air-pollution/index.md

```diff
@@ -33,7 +33,7 @@ Once in the bloodstream, PM2.5 can cause health problems to the respiratory and
                 <img src = "anatomy with annotation.svg" alt="A person breathing in PM2.5 and it entering the bloodstream and lungs"/>
             </div>
             <div class="col-md-6">
-                <img src = "sources2.svg" alt="Industrial activity, a grill, a truck, and a building demonstrate common sources of PM2.5" width="650"/>
+                <img src = "sources2.svg" alt="Industrial activity, a grill, a truck, and a building demonstrate common sources of PM2.5" width="400"/>
             </div>
```

Comment:
FYI, I'm changing `width="400"`  in here. For responsiveness, it's better to scale things as a percentage of an image's parent element - that way, instead of having a fixed with (no matter what screen size), it can have a dynamic width that adjusts for different screens (which things like a column using Bootstrap's grid usually do).